audit/projects/k8s-infra-prow-build-trusted/secrets/cncf-ci-github-token/iam.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "group:k8s-infra-prow-oncall@kubernetes.io",
+        "user:hh@ii.coop"
```

Comment:
if we plan on keeping this secret around, my preference would be to replace the `user:` with a group 

longer term we should have audit use a k8s-owned account